Expanding Space/Assets/Scripts/Travel Game/Meteorite Spawner/MeteoriteSpawner.cs

```diff
@@ -42,26 +48,38 @@ void Update()
 
         // calculates the position of where the meteorite will spawn
         spawnPositionMeterorite = GameObject.Find("BorderRight").transform.position.x + 10;
+
+        meteoriteScale = new Vector3(meteoriteSpawnScale, meteoriteSpawnScale, meteoriteSpawnScale);
     }
 
     // this function spawns a meteorite
     IEnumerator SpawnMeteorite()
     {
-        // sets the scale of the next meteorite
-        meteorite.transform.localScale = new Vector3(meteoriteSpawnScale, meteoriteSpawnScale, meteoriteSpawnScale);
+        
 
         // waits for seconds
         yield return new WaitForSeconds(3);
```

Comment:
Zouden deze 3 seconden van de WaitForSeconds ook nog in een public float kunnen?